src/NavBar.js

```diff
@@ -1,7 +1,32 @@
+import { NavLink } from "react-router-dom";
+
 function NavBar() {
+  const navStyles = {
```

Comment:
normally if you had typescript/eslint on, it would yell at you for having an empty styling block